src/components/interfaces/Camera.vue

```diff
@@ -64,4 +65,16 @@ const imgSrc = asyncComputed({
     counter: () => counter.value,
   }
 });
+
+const refreshOnSettings = debounce(() => counter.value++, 1000, {
+  leading: true,
```

Comment:
I think a 500ms leading false, trailing true would be ideal. will only pull the snapshot once and will debounce all other noisy events.